src/Components/Members/index.js

```diff
@@ -40,10 +36,6 @@ function Members() {
 
   return (
     <section>
-      <div>
```

Comment:
I think you should not delete the button because the administrator should use it

Reply:
According to the logic of the problem, the administrator should not create members, they can edit or delete them, but not create them.